test/catalogd-e2e/unpack_test.go

```diff
@@ -79,7 +79,7 @@ var _ = Describe("ClusterCatalog Unpacking", func() {
 			actualFBC, err := ReadTestCatalogServerContents(ctx, catalog, c, kubeClient)
 			Expect(err).To(Not(HaveOccurred()))
 
-			expectedFBC, err := os.ReadFile("../../testdata/catalogs/test-catalog/expected_all.json")
+			expectedFBC, err := os.ReadFile("../../catalogd/testdata/catalogs/test-catalog/expected_all.json")
```

Comment:
We cannot move now the testdata/catalogs to the root because it will broke the sync:
- https://github.com/openshift/release/blob/master/ci-operator/config/openshift/operator-framework-operator-controller/openshift-operator-framework-operator-controller-main.yaml#L26-L27